### DISCUSSION TITLE
Welcome back

### DISCUSSION THREAD

**Discussion by pacslim** [cinderblocks/radegast#16] [General]
Good to see the project and it's website is back!
just making this to say welcome back ! ;)

**Comment by cinderblocks** [comment]
Thanks! I apologize for the inconvenience, but I am not giving up on the project. I am doing the best I can!

**Comment by Leia-M** [comment]
A big welcome back from me too, Cinder! No need to apologise.  I think that I can speak for all Radegast users and say that we appreciate very much what you are doing to continue the project. Without knowing the circumstances, I had resigned to the fact that Radegast was abandonware and was wondering how long it would continue to work until LL changed something again.  Very happy to see you back, big hugs.

**Comment by GwynethLlewelyn** [comment]
Welcome back indeed! :-) I'm so glad you're still able to keep this project up and running!

Wow, I didn't even know you did host in *Ukraine*, and that the project has been accessible for so many months while the war continues to rage... I guess that the recent specific targeting of Ukraine's power infrastructure was simply too much for even the best datacentre in the world to be able to handle :-( One more reason to get rid of Putin (if there weren't enough)... _\*sigh\*_

**Comment by annitasl** [comment]
very glad to see radegast again! thanks for all your work!!
